.circleci/config.yml.j2

```diff
@@ -415,6 +417,8 @@ workflows:
             - quay.io
             - docker.io
             - software-cosign-keys
+            - acr-public
+            - acr-private
```

Comment:
Im not sure about whether we need to push to acr-private or not . But rest of the things looks good